plugins/playstation/src/Makefile.am

```diff
@@ -0,0 +1,55 @@
+plugindir = $(libdir)/gnome-games/plugins
+plugin_LTLIBRARIES = libgames-playstation-plugin.la
+
+libgames_playstation_plugin_la_DEPENDENCIES = \
+	$(top_builddir)/src/gnome-games.vapi \
+	$(NULL)
+
+libgames_playstation_plugin_la_SOURCES = \
+	playstation-header.vala \
+	playstation-plugin.vala \
+	playstation-uid.vala \
+	$(NULL)
+
+libgames_playstation_plugin_la_VALAFLAGS = \
+	--target-glib=@GLIB_MIN_VERSION@ \
+	--pkg gnome-games \
+	--pkg gio-2.0 \
+	--pkg glib-2.0 \
+	--pkg gtk+-3.0 \
+	--pkg tracker-sparql-1.0 \
+	--vapidir $(top_builddir)/src \
+	$(NULL)
+
+libgames_playstation_plugin_la_CFLAGS = \
+	$(PLAYSTATION_PLUGIN_CFLAGS) \
+	$(GNOME_GAMES_CFLAGS) \
+	-DG_LOG_DOMAIN=\"GamesDesktop\" \
```

Comment:
Not the correct log domain, it should be GamesPlayStation. The problem must be fixed in the other plugins.